sw/ground_segment/cockpit/live.ml

```diff
@@ -755,11 +763,15 @@ let create_ac = fun ?(confirm_kill=true) alert (geomap:G.widget) (acs_notebook:G
           let connect = fun ?(warning=true) setting_name strip_connect ->
             try
               let id = settings_tab#assoc setting_name in
-              strip_connect (fun x -> dl_setting_callback id x)
+              if setting_name = "kill_throttle" then
```

Comment:
still, I think it looks cleaner with a dedicated kill function that also send a setting if it exists